account_invoice_refund_link/__manifest__.py

```diff
@@ -18,6 +18,7 @@
     "post_init_hook": "post_init_hook",
     "depends": [
         'account',
+        'sale',
```

Comment:
Thanks !
Odoo solved this part of the problem https://github.com/odoo/odoo/pull/17273